src/game/basketball/results.tsx

```diff
@@ -22,114 +23,232 @@ export function ResultComponent(props: {
   const [simulationData, setSimulationData] = React.useState<
     Record<string, BasketballSimulationData>
   >({});
-  const [results, setResults] = React.useState<any[]>([]);
 
   React.useEffect(() => {
     EventSystem.on('simulationEnded', simulationEnded);
   }, []);
 
+  const chartHeight = 300;
+  const resultsWidth = 900;
+  const scoreChartWidth = resultsWidth / 2;
+  const shotsChartWidth = resultsWidth / controller.players.length;
+
+  const [insideScores, setInsideScores] = useState<number[]>([]);
```

Comment:
instead of having all the data set in different useStates, why not just merge them all into one useState and store the full simulation result there?

Reply:
Just seemed more readable this way.  Is there an issue with using multiple use states in practice?

Reply:
The data is coming from a single simulation component.

Reply:
I have combined the code to use one useState. Not quite sure the benefit of one way or the other, but possibly makes sense as all the data is related.